pkg/bitcoin/htlc/database/server.test.ts

```diff
@@ -0,0 +1,37 @@
+import server from "./server"
+import supertest from "supertest"
+import mongoose, { Mongoose } from "mongoose"
+import PostSchema from "./model/db_schema"
+
+
+describe("Test API Endpoint /", () => {
+    it("should reply to GET with default message", async () => {
```

Comment:
We should remove this test.
It shouldn't even be in the backend tests either :D